meson.build

```diff
@@ -2581,59 +2581,6 @@ meson.add_install_script(meson_make_symlink,
                          bindir / 'udevadm',
                          libexecdir / 'systemd-udevd')
 
-if conf.get('ENABLE_REMOTE') == 1 and conf.get('HAVE_LIBCURL') == 1
```

Comment:
these were all optional and not built at all without the option before, is that still the case?

Reply:
That's what the `conditions` parameter is for.

Reply:
Yes. Please see the `conditions` fields.